desktop/enterprise/release-notes.md

```diff
@@ -27,9 +27,18 @@ This version includes all the bug fixes and minor changes between Docker Desktop
 - [Docker Desktop Community release notes on Windows](/docker-for-windows/release-notes) 
 
 ### Bug fixes and minor changes
+
 - Windows: in the `admin-settings.json` configuration file, the `sharedDrives` and `sharedFolders` fields have been replaced by a unique `filesharingDirectories` field.
 - Mac: in the `admin-settings.json` configuration file, the `filesharingDirectories` has been moved out of the `linuxVM` section.
-- The `admin-settings.json` configuration file version number has been increased from 1 to 2. 
+- The `admin-settings.json` configuration file `configurationFileVersion` field value has been increased from 1 to 2.
+
+### Known issue
+
+Docker Desktop Enterprise 2.3.0.0 contains a configuration breaking change. Manual steps are required to update the `admin-settings.json` file, see the **Bug fixes and minor changes** section for the list of changes to perform.
```

Comment:
I don't like the word "breaking", it sounds too scary. I would say "contains a change in the configuration file format".

I also think we need better guidance to point to than the one above. Maybe with a recipe/example. Either in this document or another document.

Reply:
I added some step by step instructions on how to do the manual upgrade.
I tried various layouts and that one seemed like the least bad one, feel free to suggest something else!